scripts/test/IndirectCommonTests.py

```
@@ -46,7 +46,19 @@ def test_get_instrument_and_run_from_workspace(self):
 
     def test_get_instrument_and_run_failure(self):
         ws = self.make_dummy_QENS_workspace(add_logs=False)
-        self.assertRaises(RuntimeError, indirect_common.get_instrument_and_run, ws)
+        instrument, run_number = indirect_common.get_instrument_and_run(ws)
```

Ruff (0.16.2)

[warning] Unpacked variable `instrument` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)